testsys/tests/cli_test.rs

```diff
@@ -180,19 +122,7 @@ async fn test_system() {
     cmd.assert().success();
 
     // We need to give k8s some time to finalize the object creation.
-    // tokio::time::sleep(tokio::time::Duration::from_millis(100)).await;
-
-    let mut cmd = Command::cargo_bin("testsys").unwrap();
-    cmd.args(&[
-        "--kubeconfig",
-        cluster.kubeconfig().to_str().unwrap(),
-        "status",
-        "-r",
-        "never-destroy",
-        "--wait",
-    ])
-    .timeout(POD_TIMEOUT);
-    cmd.assert().success();
+    tokio::time::sleep(tokio::time::Duration::from_millis(100)).await;
```

Comment:
nit: This seems extraneous if we're waiting later in the loop anyways, could wait first in the loop before doing the assertion.